webui/src/Components/CheckboxInputField.css

```
@@ -50,7 +52,7 @@
 		width: 100%;
 		height: 100%;
 
-		color: #000;
+		color: var(--color-text-strong);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)